x-pack/solutions/observability/plugins/apm/common/service_map/types.ts

```diff
@@ -45,8 +45,8 @@ export type ConnectionType = Connection | ConnectionLegacy;
 export type DestinationType = ExitSpanDestination | ExitSpanDestinationLegacy;
 
 export interface ServiceMapConnections {
-  servicesData: ServicesResponse[];
-  anomalies: ServiceAnomaliesResponse;
+  servicesData?: ServicesResponse[];
```

Comment:
Q: Why do we change this to optional?

Reply:
That's wrong. I'll revert it. Thanks for spotting it.